i18n/en.json

```diff
@@ -2209,7 +2209,7 @@
   },
   {
     "id": "api.templates.signin_change_email.subject",
-    "translation": "[{{ .SiteName }}] You updated your sign-in method on {{ .SiteName }}"
+    "translation": "[{{ .SiteName }}] Your sign-in method has been updated"
```

Comment:
@wget the unit test will need updating to expect the modified string -- otherwise the unit test for emails will fail